### ISSUE TITLE
Tab Animation RTL Support

### ISSUE THREAD

**Issue by KAMR4N** [torgeadelin/react-native-animated-nav-tab-bar#59]
I'm using "I18nManager" to adding and force RTL (Right To Left) support for Persian language, but the View of the Animation or Active Tab is not support for RTL.

Example:
![example-Tab-bar](https://user-images.githubusercontent.com/30172779/99395938-5d75d000-28f6-11eb-9570-75a8355eee32.jpg)

**Comment by mustafakameldev** [comment]
I'm facing the same issue. could anyone give help with that?

**Comment by torgeadelin** [comment]
Will try to fix this as soon as possible.

**Comment by rayansys** [comment]
change style in index.js  line 459 to   : 

```
reactNative.I18nManager.isRTL ? {
                                right: animatedPos.interpolate({
                                    inputRange: [0, 1],
                                    outputRange: [prevPos, pos],
                                })
                                } : {
                                left: animatedPos.interpolate({
                                    inputRange: [0, 1],
                                    outputRange: [prevPos, pos],
                                })
                            }
```

**Comment by torgeadelin** [comment]
fixed, please update to 3.1.3
